db/migrate/20220427061852_add_file_structure_to_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class AddFileStructureToAssignments < ActiveRecord::Migration[6.0]
+  def change
+    add_column :assignments, :fileStructure, :string
+  end
+end
```

Comment:
This is a new parameter to store whether the user wants to process the uploaded zip file by files or by directory.